samples/config_helper_vulkan.cc

```diff
@@ -680,8 +680,10 @@ void ConfigHelperVulkan::CreateVulkanInstance() {
   instance_info.enabledExtensionCount = 1U;
   instance_info.ppEnabledExtensionNames = &kExtensionForValidationLayer;
 
-  assert(vkCreateInstance(&instance_info, nullptr, &vulkan_instance_) ==
-         VK_SUCCESS);
+  if (vkCreateInstance(&instance_info, nullptr, &vulkan_instance_) !=
+      VK_SUCCESS) {
+    assert(false && "Sample: vkCreateInstance fail");
```

Comment:
This needs to return an error. In Release mode, the code will just continue as though everything is fine.

Reply:
done